client-lib/java/src/main/java/com/abcxyz/jvs/Justification.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.abcxyz.jvs;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+@JsonDeserialize(using = JustificationDeserializer.class)
+public class Justification {
```

Comment:
This is fine. But I wonder if we _should_ use the proto generated `Justification` type instead?

Reply:
I'm not super convinced we should be embedding the protos directly in Go either. I think we should have auxiliary structures here.

Reply:
(Not blocking)
Yea, you mentioned it in an earlier PR. I wonder what's the purpose of having an auxiliary structure?